public/js/csv-generator.js

```diff
@@ -0,0 +1,13 @@
+const csvGenerator = {
+	init() {
```

Comment:
i think this module name could be more clear, what do you think of `downloadCsvButton.js` instead